### DISCUSSION TITLE
Use a msg.var or mustache template inside get history node for field relativeTime

### DISCUSSION THREAD

**Discussion by klatka** [zachowj/node-red-contrib-home-assistant-websocket#1279] [Feature Request]
![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/42534401/548905b6-3dd6-422d-bee8-828ce19f9632)

The mustache template in the field `in the last` does not work.

Proposed change:
Change get history node to make it work

**Comment by zachowj** [comment]
You can set `relative time` from message inputs. <https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/get-history.html#relativetime>

Change the location of `msg.relative_time_string` to `msg.payload.relativeTime` and leave the text field empty in the node.

**Reply by klatka** [reply]
Thanks, I overlooked the first sentence in the documentation. I already use payload for something else. I need to back up the payload before using the get history node and then set it back to its original value. A mustache template would be easier - maybe a suggestion in the future.

**Comment by klatka** [comment]
I found that the attributes `last_changed` and `last_updated` are set to the relative time even when there was no action.

This is the actual history of the entity:
<img width="559" alt="image" src="https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/42534401/374d795b-0c3b-472c-a895-a798973032f6">

If I set relativeTime to 1 hour I get one result showing the state `off` which is true. But the attributes show the last action was one hour ago which is wrong. This might be a bug or should be documented.
